### ISSUE TITLE
Time is not allowed in filters.

### ISSUE THREAD

**Issue by rashidabbas786** [sflagp/QBXML-Object-Models#3]
Hello Greg,

I have been busy with a few other things and wasn't able to test the previous build. During my work, while applying filters on invoices on ModifiedDate, I found that it never parses the time part from a DateTime to XML but only parses the date. 
The expected date format is: 
`yyyy-MM-ddTHH:mm:sszzz
`

Currently implemented as below: 

<img width="436" alt="image" src="https://github.com/sflagp/QBXML-Object-Models/assets/81164526/9d234306-1121-43b4-94a5-7522bac9fea2">

**Comment by rashidabbas786** [comment]
Again I have made an Extension to handle this:

```
using QbModels;
using System.Xml.Linq;
namespace QBSync.Extensions
{
    public static class InvoiceQueryRqExtensions
    {
        public static string ToXMLString(this InvoiceQueryRq rq)
        {
            string xml = rq.ToString();

            if (rq.ModifiedDateRangeFilter != null)
            {
                XDocument document = XDocument.Parse(xml);
                ModifiedDateRangeFilterDto filter = rq.ModifiedDateRangeFilter;
                if (filter.FromModifiedDate != null)
                {
                    document.Descendants("FromModifiedDate").First().Value = ((DateTime)filter.FromModifiedDate).ToString("yyyy-MM-ddTHH:mm:sszzz");
                }
                if (filter.ToModifiedDate != null)
                {
                    document.Descendants("ToModifiedDate").First().Value = ((DateTime)filter.ToModifiedDate).ToString("yyyy-MM-ddTHH:mm:sszzz");
                }

                xml = document.ToString();


            }
            return xml;
        
        }

    }
}


```

**Comment by sflagp** [comment]
Thanks for the testing and feedback. Took me a second but I see what you're saying about the date/time in modified date range. Here is an updated version of the file with the correct modified date range format. I had published v1.2.5 last night so it's now v.1.2.6. I should have bumped up the version to v1.3 with the new features for the batch requests but oh well.

[QbModels.1.2.6-beta03.zip](https://github.com/sflagp/QBXML-Object-Models/files/13743316/QbModels.1.2.6-beta03.zip)

**Comment by rashidabbas786** [comment]
Hello Greg,

Thank you so much <3
